rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -922,6 +919,10 @@ def _update_participant_summary(
             if race != participant_summary.race:
                 participant_summary.race = race
                 something_changed = True
+            if not participant_summary.aian \
+                    and Race.AMERICAN_INDIAN_OR_ALASKA_NATIVE in get_all_races_from_codes(race_codes):
+                participant_summary.aian = True
+                something_changed = True
```

Comment:
So if a participant answers that they're AIAN and then later we get an updated answer where they don't say AIAN, then we'll still keep the AIAN flag set to True?